documentation/specs/proposed/MSBuild_Server_onepager.md

```diff
@@ -0,0 +1,66 @@
+## MSBuild Server
+
+MSBuild server aims to create a persistent entry node for the MSBuild
+that we would communicate with via a thin client. We want to get from
+the current state of “spawn a complete process for every CLI invocation”
+to “we have a server process in the background and we only spawn a small
+CLI handler that will tell the server what to build”.
```

Comment:
It would be good to add that this is not a completely new implementation, but re-enablement of a previous work. I had realized this only after reading the Cost and Plan sections.